examples/public_only/main.tf

```diff
@@ -0,0 +1,17 @@
+module "vpc" {
+  source = "../.."
```

Comment:
Ditto other- replace with tf registry source, add a README, and recommend adding ">= 0.0.1" version